File: integrations/src/main/java/services/voice/dto/CallLogRequest.java
package services.voice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CallLogRequest {
    private String callSid;
    private String entityName;
    private Long entityId;
    private String direction;
    private String fromNumber;
    private String toNumber;
    private String status = "initiated";
    private String exophone;
}

File: integrations/src/main/kotlin/services/voice/dto/CallLogRequest.kt
package services.voice.dto

data class CallLogRequest(
    val callSid: String,
    val entityName: String,
    val entityId: Long,
    val direction: String,
    val fromNumber: String,
    val toNumber: String,
    val status: String = "initiated",
    val exophone: String? = null
)
